src/components/Input/__tests__/__snapshots__/Input.spec.js.snap

```diff
@@ -198,21 +198,20 @@ textarea.c2:hover {
 }
 
 <div
-  class="text--input-regular text--input-top c0"
+  class="text--input-undefined text--input-undefined c0"
```

Comment:
@jorge-benjumea-globant Hmm what caused these snapshot changes? 🤔

Reply:
@mlampedx I thought it was for the node version, but just cleaned a run the test script with 8.11.1. and the snapshot persists, it seems to bee the forwardRef wrapper made in the Input component

https://github.com/ticketmaster/aurora/pull/592/commits/09510a0983071b219b9bc47d3490c9a529ab09f8#diff-8030fbcffcceb6c2680e6536fb739561R91

Reply:
Done, it was the allocation of `forwardRef` wrapper

Reply:
Aha! Good find 😄